librosa/display.py

```diff
@@ -617,6 +618,19 @@ def __envelope(x, hop):
     return x_frame.max(axis=1)
 
 
+_categorical_axis_types = ['chroma', 'chroma_h', 'tonnetz', 'frames']
+_time_axis_types = ['linear', 'fft', 'hz', 'log,' 'fft_note', 'fft_svara', 'mel', 'cqt_hz', 'cqt_note', 'cqt_svara']
```

Comment:
These are frequency-like axes, not time-like.  I think the equivalence class is too broad here though, and should be reduced to the following:

- linear, fft, hz, fft_note, fft_svara
- log
- mel
- cqt_hz, cqt_note, cqt_svara

Since singletons (log and mel) are silly to include, we could shortcut this in the check by saying `x_axis == y_axis or (x_axis, y_axis) in _AXIS_COMPAT`.  This way we won't have to clutter the object too much.